cumulusci/tasks/salesforce/tests/test_insert_record.py

```diff
@@ -0,0 +1,40 @@
+from unittest import mock, TestCase
+
+from cumulusci.tasks.salesforce.insert_record import InsertRecord
+from cumulusci.core.exceptions import TaskOptionsError
+from .util import create_task
+
+
+class TestCreateRecord(TestCase):
+    def test_run_task(self):
+        task = create_task(
+            InsertRecord,
+            {
+                "object": "PermissionSet",
+                "values": "Name:HardDelete,PermissionsBulkApiHardDelete:true",
+            },
+        )
+        create = mock.Mock()
+        task.sf = mock.Mock(create=create)
+        task.sf.PermissionSet.create.return_value = {
+            "id": "0PS3D000000MKTqWAO",
+            "success": True,
+            "errors": [],
+        }
+
+        task._run_task()
+        task.sf.PermissionSet.create.assert_called_with(
+            {"Name": "HardDelete", "PermissionsBulkApiHardDelete": "true"}
+        )
+
+    def test_syntax_errors(self):
+        task = create_task(
+            InsertRecord,
+            {
+                "object": "PermissionSet",
+                "values": "Name:Hard:Delete,PermissionsB:ulkApiHardDelete:true",
+            },
+        )
+        task._init_task()
+        with self.assertRaises(TaskOptionsError):
```

Comment:
If you use `pytest.raises` here then the test class doesn't need to extend unittest.TestCase.

Reply:
True but then I'd have to import pytest.

Also: 14 of our files use pytest.raises and 42 use self.assertRaises.

Reply:
Yes, we've been switching them over to pytest as we work on them.

Reply:
Okay but why? It kind of rubs me the wrong way to use identical functionality from a 3rd party package instead of the standard library unless there is a good reason.

Reply:
That's fair -- I think they are pretty identical functionality-wise, and this is partly why I've felt no particular need to switch everything over to the pytest version. However there are some pytest features that can't be used in unittest.TestCase subclasses (https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses), and then it's handy to have the alternative.